libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Scopes/BotStateMemoryScope.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Bot.Builder.Dialogs.Memory.Scopes
     /// <summary>
     /// BotStateMemoryScope represents a BotState scoped memory.
     /// </summary>
-    /// <remarks>This relies on the BotState object being accessible from turnContext.TurnState.Get&lt;T&gt().</remarks>
+    /// <remarks>This relies on the BotState object being accessible from turnContext.TurnState.Get&lt;T&amp;gt().</remarks>
```

Comment:
**&lt** is the 'less than' symbol `<`, **&amp** is the ampersand `&`. Is **gt** meant to be **&gt**, a.k.a. _the closing 'greater than' symbol_?